File: tests/fixtures/sample_undocumented.py
class PythonSample:
    def undocumented_method(self, x: int, y: str):
        pass
        
    def documented_method(self):
        """This is documented."""
        return True

def undocumented_function(a, b):
    print(a, b)

def documented_function():
    """Has a docstring."""
    pass

File: tests/fixtures/sample_undocumented.js
class JSSample {
    undocumentedMethod(x, y) {
        console.log(x, y);
    }

    /** JSDoc is here */
    documentedMethod() {
        return true;
    }
}

function undocumentedFunction(a, b) {
    console.log(a, b);
}

/** Has JSDoc */
function documentedFunction() {
    pass;
}

const arrowFunc = (z) => {
    return z * 2;
};
